src/engine/effects/message.h

```diff
@@ -88,9 +90,11 @@ struct EffectsRequest {
     union {
         struct {
             EngineEffectRack* pRack;
+            bool preFader;
```

Comment:
Both values _pre_ and _post_ should have the same significance. Type boolean is therefore not an appropriate choice and also doesn't guarantee type safety. Please use _enum class AuxSend { PreFader, PostFader }_ for this purpose. I'm not sure about the appropriate name of this new type.

Reply:
Ping!

Reply:
Any ideas for names for this enum class? My first thought was "FaderOrientation", but that is too easy to confuse with crossfader orientation. Maybe "FaderStage"? Or "SignalProcessingStage"?